aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -352,6 +353,16 @@ module aptos_framework::staking_config {
         staking_config.voting_power_increase_limit = new_voting_power_increase_limit;
     }
 
+    public(friend) fun lock() acquires StakingConfig {
```

Comment:
Where are they used?

Reply:
This is currently not used. This was added back when @movekevin suggested we use an existing flag `0x1::staking_config::StakingConfig.allow_validator_set_change` to represent validator set locking status.

But a new flag `0x1::config_for_next_epoch::ValidatorSeChangeLocked` still seems cleaner to me... And that's what the current PR is using.

@movekevin wdyt?

Reply:
it can cause several uts to fail, maybe not touch it?